types/react-virtualized-select/tsconfig.json

```diff
@@ -5,6 +5,9 @@
             "es6",
             "dom"
         ],
+        "paths": {
+            "react": ["react/v17"]
```

Comment:
It should be `v16` but the definitions-parser does not support this: https://github.com/microsoft/DefinitelyTyped-tools/issues/423